warehouse/query-core/src/test/java/datawave/query/UniqueTest.java

```diff
@@ -298,12 +313,14 @@ public void testUniquenessUsingLuceneFunction() throws Exception {
         runTestQueryWithUniqueness(expected, queryString, startDate, endDate, extraParameters);
 
         queryString = "UUID:/^[CS].*/ AND #UNIQUE(DEATH_DATE,$BIRTH_DATE)";
+        expected.clear();
         expected.add(Sets.newHashSet(WiseGuysIngest.sopranoUID));
         expected.add(Sets.newHashSet(WiseGuysIngest.corleoneUID));
         expected.add(Sets.newHashSet(WiseGuysIngest.caponeUID));
         runTestQueryWithUniqueness(expected, queryString, startDate, endDate, extraParameters);
 
         queryString = "UUID:/^[CS].*/ AND #UNIQUE(death_date,birth_date)";
+        expected.clear();
```

Comment:
Instead of clearing the collection here, could clear the collection at the end of the driver method

Reply:
probably.  I will take this into consideration.

Reply:
That would be a side effect and hence not obvious to the user.  I would rather leave the explicit clear calls where they are.